core/internal/helpers/sarama.go

```diff
@@ -205,6 +205,9 @@ type SaramaClient interface {
 	// NewConsumerFromClient creates a new consumer using the given client. It is still necessary to call Close() on the
 	// underlying client when shutting down this consumer.
 	NewConsumerFromClient() (sarama.Consumer, error)
+
+	// List the consumer groups available in the cluster.
+	ListConsumerGroups() (map[string]string, error)
```

Comment:
Why the value type of the map is `string`?

Reply:
something related the return type of Sarama, as it returns a Map with the consumer group and the consumer group type; here, we only care about if the consumer group exists, a `Set` would be great, but go doesn't have it, so we still need a `Map`, but we don't care about the value, so I could convert the type to `map[int]struct{}`, I decided not to do it as there is no performance gains, and it'll be more wasteful to create another map just to get rid of the undesired value